refarch-apigateway/.gitattributes

```diff
@@ -0,0 +1,2 @@
+# Alle Textdateien (* text=auto) sollen den Unix-Zeilenumbruch (eol=lf) nutzen
+* text=auto eol=lf
```

Comment:
I guess this file was added due to itm-prettier-codeformat. However, it should also affect Java files here. That's not what we want. Maybe we can remove the file completely?

Reply:
its needed because everyhone has their own git configuration. With that file it is guaranteed that everyone uses `lf` as default line break. Removing it could cause problems.

Reply:
Makes sense. But shouldnt we add the file to all other templates as well? I will create an issue for it.

Reply:
It might be even better to centralize this file at the repository root. We might be able to do the same with the .gitignore file.